### PR TITLE
Wqp 1439 - Add in afterLoad flow, tests, and end to end test.

### PR DESCRIPTION
**Pull request by mbucknell**

### CODE REVIEW

File: src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/nwisWqxMediumCd/nwisWqxMediumCd.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/nwisWqxRptLevCd/nwisWqxRptLevCd.xml")    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/parm/parm.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/parmMeth/parmMeth.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/qwResult/qwResult.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/qwSample/qwSample.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/sitefile/sitefile.xml")
+    @DatabaseSetup(connection = CONNECTION_NWIS, value = "classpath:/testData/nwis/siteTp/siteTp.xml")
+
+    //Tables
+    //@ExpectedDatabase(
```
Comment (mbucknell) [inline src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java]: This is the test data that I couldn't get to work. When I do what I think is the query, I get 58 rows but the test says it is getting back 62.

File: src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    public static final String EXPECTED_DATABASE_QUERY_FOREIGN_KEY = BASE_EXPECTED_DATABASE_QUERY_FOREIGN_KEY
+            + " like '%_nwis'";
+
+    @Autowired
+    @Qualifier("nwisToWqpFlow")
+    private Flow nwisToWqpFlow;
+
+    @Test
+    // Lookups and mysql data tables
+    @DatabaseSetup(value = "classpath:/testResult/wqp/monitoringLocation/empty.xml")
```
Comment (dsteinich) [inline src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java]: Not needed in the end-to-end. Existing swap table is destroyed and recreated in the setup flow.

File: src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java (hunk 1 above)
Comment (dsteinich) [reply src/test/java/gov/acwi/wqp/etl/EtlNwisToWqpIT.java]: not sure if including
and table_name not like '%swap%'
in the query would help

**Comment by mbucknell** [comment]
I did try the install query as you suggested and it did not matter. I will leave commented out for now.

**Pull request merged**